### PR TITLE
Polish the 'sops help' output a bit

### PR DESCRIPTION
**Pull request by felixfontein**
In particular re-adjust line lengths.

Also mention that flags should always come before filenames.

### CODE REVIEW

File: cmd/sops/main.go [hunk 1]
```diff
@@ -80,44 +81,51 @@ func main() {
...
 
    To use a different GPG binary than the one in your PATH, set SOPS_GPG_EXEC.
 
    To select a different editor than the default (vim), set EDITOR.
 
-   For more information, see the README at github.com/mozilla/sops`
+   Note that flags must always be provided before the filename to operate on.
+   Otherwise, they will be ignored.
+
+   For more information, see the README at https://github.com/mozilla/sops`
```
Comment (hiddeco) [inline cmd/sops/main.go:128]:
```suggestion
   For more information, see the README at https://github.com/getsops/sops`
```

Comment (felixfontein) [reply cmd/sops/main.go]: Thanks! I totally missed that.

**Review by hiddeco — APPROVED** [review]
Polishing PRs are my favorite :1st_place_medal:

**Comment by felixfontein** [comment]
@hiddeco thanks for reviewing this!

**Pull request merged**